### ISSUE TITLE
508-defect-4 [COGNITION]: Breadcrumbs should match `h1`

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#29759]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

We recently updated a couple of the search page's `h1`s from "Search results" to "Search" in order to communicate to screen reader users that they can both search and browse search results. However, we did not update the breadcrumbs to match those changes which may be confusing or inconsistent to users.

## Acceptance Criteria

- [ ] #29733 is closed 
- [ ] Breadcrumbs for VA.gov and Resources and support match their respective `h1`s

### Before
<img width="784" alt="Screen Shot 2021-09-08 at 2 28 44 PM" src="https://user-images.githubusercontent.com/14154792/132566179-1024cb97-0f83-4140-af98-8da7338ab976.png">

<img width="800" alt="Screen Shot 2021-09-08 at 2 38 23 PM" src="https://user-images.githubusercontent.com/14154792/132566229-88452e9e-1de2-430a-8116-ccee3a3d1799.png">

### After
<img width="787" alt="Screen Shot 2021-09-08 at 2 28 37 PM" src="https://user-images.githubusercontent.com/14154792/132566193-a383e7f3-818d-4707-95f6-b1bbc5e136f0.png">

<img width="791" alt="Screen Shot 2021-09-08 at 2 30 42 PM" src="https://user-images.githubusercontent.com/14154792/132566157-9e5635b4-864e-4898-b043-b0b3bd3c08e0.png">

**Comment by denisecoveyduc** [comment]
@srsuddath there is a portion of this that is for our va.gov global page. Could you resolve?

**Comment by srsuddath** [comment]
I have added a fix for this, and will push up a PR shortly to resolve S&D's portion of this!

**Comment by laflannery** [comment]
Global search breadcrumbs have been updated.  The remaining task here is to update the breadcrumb on the [Resources and Support search results page](https://www.va.gov/resources/search/?query=disability).

- The H1 of the R&S search results page is "Resources and Support Search Results"
- The breadcrumb that currently says "Search results" should be updated to match the H1
![image](https://user-images.githubusercontent.com/106678594/229222380-f4f6a2cd-7dd4-4d18-9c67-2fb29425f618.png)
